agent-shell.el

```diff
@@ -258,6 +258,14 @@ See `agent-shell-*-make-*-config' for details."
   :type '(repeat (alist :key-type symbol :value-type sexp))
   :group 'agent-shell)
 
+(defcustom agent-shell-new-shell-config nil
```

Comment:
Can we name this agent-shell-preferred-agent-config, please?

Reply:
done!